api_connect_widget.go

```diff
@@ -0,0 +1,143 @@
+ *
+ * API version: 0.1
+ */
+
+package atrium-go
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"fmt"
+)
+
+// Linger please
+var (
+	_ context.Context
+)
+
+type ConnectWidgetApiService service
+
+/* 
+ConnectWidgetApiService Embedding in a website
+This endpoint will return a URL for an embeddable version of MX Connect.
+ * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+ * @param userGuid The unique identifier for a &#x60;user&#x60;.
+ * @param body Optional config options for WebView (is_mobile_webview, current_institution_code, current_member_guid, update_credentials)
+
+@return ConnectWidget
+*/
+func (a *ConnectWidgetApiService) GetConnectWidget(ctx context.Context, userGuid string, body ConnectWidgetRequestBody) (ConnectWidget, *http.Response, error) {
+	var (
+		localVarHttpMethod = strings.ToUpper("Post")
+		localVarPostBody   interface{}
+		localVarFileName   string
+		localVarFileBytes  []byte
+		localVarReturnValue ConnectWidget
+	)
+
+	// create path and map variables
+	localVarPath := a.client.cfg.BasePath + "/users/{user_guid}/connect_widget_url"
+	localVarPath = strings.Replace(localVarPath, "{"+"user_guid"+"}", fmt.Sprintf("%v", userGuid), -1)
+
+	localVarHeaderParams := make(map[string]string)
+	localVarQueryParams := url.Values{}
+	localVarFormParams := url.Values{}
+
+	// to determine the Content-Type header
+	localVarHttpContentTypes := []string{"application/json"}
+
+	// set Content-Type header
+	localVarHttpContentType := selectHeaderContentType(localVarHttpContentTypes)
+	if localVarHttpContentType != "" {
+		localVarHeaderParams["Content-Type"] = localVarHttpContentType
+	}
+
+	// to determine the Accept header
+	localVarHttpHeaderAccepts := []string{"application/vnd.mx.atrium.v1+json"}
+
+	// set Accept header
+	localVarHttpHeaderAccept := selectHeaderAccept(localVarHttpHeaderAccepts)
+	if localVarHttpHeaderAccept != "" {
+		localVarHeaderParams["Accept"] = localVarHttpHeaderAccept
+	}
+	// body params
+	localVarPostBody = &body
+	if ctx != nil {
+		// API Key Authentication
+		if auth, ok := ctx.Value(ContextAPIKey).(APIKey); ok {
+			var key string
+			if auth.Prefix != "" {
+				key = auth.Prefix + " " + auth.Key
+			} else {
+				key = auth.Key
+			}
+			localVarHeaderParams["MX-API-Key"] = key
+			
+		}
+	}
+	if ctx != nil {
+		// API Key Authentication
+		if auth, ok := ctx.Value(ContextAPIKey).(APIKey); ok {
```

Comment:
Looks to be using the same key (`ContextAPIKey`) for both the API key and the Client ID values.